index.html

```diff
@@ -1321,12 +1294,85 @@
       <p>
         The following example shows how to represent US$55.00.
       </p>
-      <pre class="example">
-{
-  "currency": "USD",
-  "value" : "55.00"
-}
+      <pre class="example js">
+        {
+          "currency": "USD",
+          "value" : "55.00"
+        }
       </pre>
+      <section>
+        <h3>
+          Validity checkers
+        </h3>
+        <p>
+          A <dfn data-cite="INFRA#javascript-string">JavaScript string</dfn> is
+          a <dfn>valid decimal monetary value</dfn> if it consists of the
+          following <dfn data-lt="code point" data-cite="INFRA#code-point">code
+          points</dfn> in the given order: [[!INFRA]]
+        </p>
+        <ol>
+          <li>Optionally, a single U+002D HYPHEN-MINUS (-), to indicate that
+          the amount is negative.
```

Comment:
In Infra we've moved toward omitting the spelled out name, so you'd just do `U+002D (-)` these days. See https://infra.spec.whatwg.org/#code-points